### DISCUSSION TITLE
How to change font colour?

### DISCUSSION THREAD

**Discussion by fragmentsofthought** [Akifyss/obsidian-border#294] [Q&A]
Hey there, I'm pretty new to the plugin, but I'm just trying to work out if there is a way to change the default font colour in the editor for this theme?

**Comment by Akifyss** [comment]
You can download the Style Settings plugin and adjust font colors in this section:
![image](https://github.com/Akifyss/obsidian-border/assets/23567154/ba70a177-83ba-4682-a569-de1bcfe59bbc)

**Reply by fragmentsofthought** [reply]
Is it possible to edit the colour for just the editor page?

**Reply by Akifyss** [reply]
Sorry, at the moment it is not possible to modify only the font color of the editor.
